netci.groovy

```diff
@@ -658,35 +658,52 @@ branchList.each { branchName ->
     }
 }
 
-// Generate the build and test versions for Windows_NT.  When full build/run is supported on a platform, those platforms
+// Generate the build and test versions for Windows_NT and Ubuntu.  When full build/run is supported on a platform, those platforms
 // could be removed from above and then added in below.
-def supportedFullCyclePlatforms = ['Windows_NT']
+def supportedFullCyclePlatforms = ['Windows_NT', 'Ubuntu']
```

Comment:
For example, we pass this through to "setMachineAffinity" in some cases, and I'm not sure what that does exactly. It might be expecting "Ubuntu" rather than "Ubuntu14.04"